README.md

```diff
@@ -310,6 +312,8 @@ consumer0, err = s.CreateConsumer("<consumer-name>",
 // creation from a Conn
 consumer1, err = c.CreateConsumer("<station-name>", "<consumer-name>", ...) 
 ```
+Note:
+When consuming from a station with more than one partition, the consumer will consume messages in Round Robin from the partitions.
```

Comment:
... will consume messages in Round Robin fashion from the different  partitions.